src/entity/cardDeck.test.js

```diff
@@ -0,0 +1,28 @@
+import { cardDeck, shuffleDeck } from "./cardDeck";
+import { getArrayItems } from "../utils/arrayItems";
+
+describe("cardDeck", () => {
+  it("should return the created deck", () => {
+    expect(cardDeck).toMatchSnapshot();
+  });
+});
+
+describe("shuffleDeck", () => {
+  it("should return a shuffled deck", () => {
+    expect(cardDeck).not.toStrictEqual(shuffleDeck(cardDeck));
```

Comment:
Este teste é não cobre 100% do comportamento esperado de shuffle.
Edge cases:
- se `cardDeck` for um array vazio o teste falha.
- se `cardDeck` possuir apenas um elemento o teste falha
- existe uma chance da função `shuffleDeck` retornar o array na mesma ordem do original, já que ela usa `random`. Quanto menor for o array de entrada, maior essa chance.